### DISCUSSION TITLE
Please add features to select default font in newer release.

### DISCUSSION THREAD

**Discussion by medukia** [sumatrapdfreader/sumatrapdf#3348] [Ideas]
The default font(probably times roman) used for the document(especially Epub format) with no embedded fonts doesn't look very beautiful(especially when rendering asian glyphs like chinese, japanese or korean).

I always remove embedded fonts for the ebooks to take up less storage space.
I might be able to change the windows' default system fonts but it's killing a fly with sledgehammer! Not very desirable solution.

Please let us choose our fonts for the better reading experience, like previous build.

Many thanks.

**Comment by robot3498712** [comment]
I fully agree. Changing font and background color is essential. I'll stick with v3.3.3 for the time being.

> doesn't look very beautiful

Well, it looks like shit and I won't spend a single minute reading such a file in Sumatra.

**Comment by MengYufei** [comment]
Yes, defaut font is difficult to accept.
![2024-07-04_090026](https://github.com/sumatrapdfreader/sumatrapdf/assets/44423204/770980bd-1b60-4a73-a07f-567012639803)

**Comment by dobelachaos** [comment]
希望能够再增加自定义字体功能
